projects/experimental/components/compass/compass.component.html

```diff
@@ -0,0 +1,27 @@
+<div class="t-compass">
```

Comment:
This nesting is unnecessary, we can have host and inside it a single SVG element for the beak. If we do not care about the beak being slightly round we can even have no nesting at all and use pseudo element:

:host - a circle of current color
::before - a triangle beak of current color
::after - a circle with border and background gradient to apply tint (see my comment in styles)

::after should inherit border-color from :host so people can override it to be black when necessary

Reply:
fixed